### PR TITLE
WAI-41 Average Cohort Survival

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Issue #:
Resolves WAI-41
NZ-41

### Changes:

- Add an attribute type to non-target districts
- Add a new HLO1 list visual for primary education at country level
- Add indicators to calculate average vs target cohort survival rate
- Add a drilldown graph

---

### Screenshots:
See attached ticket

### CODE REVIEW

**Review by edmofro — CHANGES_REQUESTED** [review]
Before I review the indicators etc., could you please update the report to use the [new config](https://beyond-essential.slab.com/posts/report-server-transform-config-design-change-proposals-k8l3tch0) that just got merged to dev? Cheers!

File: packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js
Comment (EMcQ-BES) [inline packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js:407]: Just realised that I only actually added this dashboard in WAI-44 so while it was working on my local it actually relies on [another PR](https://github.com/beyondessential/tupaia/pull/3171)

I'm just going to wait until that PR gets merged so I don't have to do complicated git stuff

File: packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js
Comment (edmofro) [inline packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js:98]: And fair enough too

File: packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js
Comment (edmofro) [inline packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js:169]:
```suggestion
```
Default is now `single`, which adds the extra assertion that we are only expecting one to be defined

File: packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js
Comment (edmofro) [inline packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js:212]: Same deal

**Review by edmofro — APPROVED** [review]
Couple of suggestions but lgtm!

**Review by edmofro — APPROVED** [review]

**Pull request merged**
